.github/workflows/style.yml

```
@@ -0,0 +1,53 @@
+---
+name: Code Style
+on:
+  - push
+  - pull_request
+jobs:
+  isort:
```

We have the issue of this workflow running twice when PRs are created from this repo. This if clause will avoid this running twice by not running the workflow if the PR is from the same repo.

```suggestion
  isort:
     if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
```

Does anyone have a better idea? Maybe @margau ?

Sources:

- <https://github.community/t/duplicate-checks-on-push-and-pull-request-simultaneous-event/18012/5>
- <https://github.com/corejam/corejam/pull/7>

---

Added in 60cfee9